main.py

```diff
@@ -0,0 +1,16 @@
+# this allows us to use code from
+# the open-source pygame library
+# throughout this file
+import pygame
+
+
+def main():
+    try:
+        pygame.init()
+        print('Starting asteroids!')
+        pygame.quit()
```

Comment:
**Suggestion:** Ensure `pygame.quit()` is called only when Pygame is initialized.

**Existing Code:**
```Python
pygame.quit()
```

**Improved Code:**
```Python
if pygame.get_init():
    pygame.quit()
```

**Details:**
Ensure `pygame.quit()` is called only when Pygame has been successfully initialized.